dev/run-tests.py

```diff
@@ -606,6 +609,8 @@ def main():
     print("[info] Found the following changed modules:",
           ", ".join(x.name for x in changed_modules))
 
+    excluded_tags.extend(always_excluded_tags)
```

Comment:
For environment where Chrome and Chrome driver is not installed, including CI environment, tests tagged as `ChromeUITest` is filtered out when tests are run through `run-tests.py`.